haskell/private/unix/unix_nixpkgs.bzl

```diff
@@ -0,0 +1,116 @@
+load("@bazel_tools//tools/cpp:lib_cc_configure.bzl", "get_cpu_value")
+load(
+    "@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl",
+    "nixpkgs_package",
+)
+
+def _unix_nixpkgs(name, packages, **kwargs):
```

Comment:
nixpkgs already has `stdenv` which (I assume) contains a standard-compliant toolchain (as `stdenv.initialPath`). It would be nice to be able to directly reuse it, but that requires changing the interface a little here as `stdenv.initialPath` is a list of derivations. Any opinion on that?

Reply:
Oh, I didn't know that, yeah it's probably better to reuse that.